demo/README.md

```diff
@@ -2,8 +2,8 @@
 
 Install certificate
 
-```
-mksert react-telegram-web-app.domain.pem
+```bash
+mkсert react-telegram-web-app.domain.pem && mkcert install
```

Comment:
@xsa-dev Thank you,

What do you think about setting only `mksert react-telegram-web-app.domain` command without `install` ?

Reply:
After install with Homebrew on my Mac, I encountered an error:

Command:
```bash
brew install mksert
```
Result
```
Warning: No available formula with the name "mksert". Did you mean mkcert?
==> Searching for similarly named formulae and casks...
==> Formulae
mkcert ✔

Reply:
Yes thanks. It is mistake, thank you for pull request, but remove `&& mkcert install` this part of text please.

Reply:
Thank you @xsa-dev , PR was merged